lightning/src/ln/channel.rs

```diff
@@ -1541,6 +1543,10 @@ pub(super) struct ChannelContext<SP: Deref> where SP::Target: SignerProvider {
 	/// If we can't release a [`ChannelMonitorUpdate`] until some external action completes, we
 	/// store it here and only release it to the `ChannelManager` once it asks for it.
 	blocked_monitor_updates: Vec<PendingChannelMonitorUpdate>,
+
+	/// The current interactive transaction construction session under negotiation.
+	#[cfg(dual_funding)]
+	interactive_tx_constructor: Option<InteractiveTxConstructor>,
```

Comment:
This could always still be populated in an ordinary `Channel` too because of splicing. In fact we keep it around after promoting inbound and outbound V2 channels to `Channel` since we still need to complete the interactive signing at that point.

I could triplicate this in `InboundV2Channel`, `OutboundV2Channel`, and `Channel` if that makes more sense, and then the V1 unfunded channels wouldn't have access to it. I'm mostly indifferent to that. So if it's preferred I'll do that! :)